packages/kbn-rule-data-utils/src/alerts_as_data_rbac.ts

```diff
@@ -91,3 +93,8 @@ export const getEsQueryConfig = (params?: GetEsQueryConfigParamType): EsQueryCon
   }, {} as EsQueryConfig);
   return paramKeysWithValues;
 };
+
+/**
+ * TODO: Abstract it and remove it
+ */
+export const isSiemRuleType = (ruleTypeId: string) => ruleTypeId.startsWith('siem.');
```

Comment:
In the codebase, we have a lot of checks (hacks) related to security rule types. To reduce the scope of the PR as much as possible I choose to try to fix it slowly on subsequent PRs. At the moment is needed.